### PR TITLE
1422 overlay overview: get clipped thumbnail by position

### PR DESCRIPTION
**Pull request by ramtob**

### CODE REVIEW

File: src/app/@ansyn/ansyn/app-effects/effects/overlays.app.effects.ts [hunk 1]
```diff
@@ -182,6 +195,20 @@ export class OverlaysAppEffects {
 			return new DisplayOverlayAction({ overlay, mapId });
 		});
 
+	/**
+	 * @type Effect
+	 * @name setHoveredOverlay$
+	 * @action SetHoveredOverlayAction
+	 */
+	@Effect()
+	setHoveredOverlay$: Observable<any> = this.store$.select(selectDropMarkup)
```
[the remark this answers is not in the document]
Comment (ramtob) [reply src/app/@ansyn/ansyn/app-effects/effects/overlays.app.effects.ts]: Not here

File: src/app/@ansyn/ansyn/app-effects/effects/overlays.app.effects.ts [hunk 2]
```diff
@@ -182,10 +200,61 @@ export class OverlaysAppEffects {
...
+		if (!communicator) {
+			return Observable.of([overlay, null])
+		}
+		return communicator.getPosition().map((position) => [overlay, position])
+	});
+	private getOverlayWithNewThumbnail = mergeMap(([overlay, position]: [Overlay, CaseMapPosition]) => {
+		if (!overlay) {
+			return [overlay];
+		}
+		const overlaySourceProvider = this.getOverlaySourceProvider(overlay.sourceType);
```
Comment (yairtawil) [inline src/app/@ansyn/ansyn/app-effects/effects/overlays.app.effects.ts]: rename

File: src/app/@ansyn/overlays/components/overlay-overview/overlay-overview.component.ts [hunk 3]
```diff
@@ -36,11 +36,9 @@ export class OverlayOverviewComponent implements OnInit, OnDestroy {
 	@HostBinding('style.left.px') left = 0;
 	@HostBinding('style.top.px') top = 0;
 
-	hoveredOverlay$: Observable<any> = this.store$.select(selectDropMarkup)
-		.withLatestFrom((this.store$.select(overlaysStateSelector).pluck<IOverlaysState, Map<any, any>>('overlays')))
-		.map(([markupMap, overlays]: [ExtendMap<MarkUpClass, MarkUpData>, Map<any, any>]) =>
-			overlays.get(markupMap && markupMap.get(MarkUpClass.hover).overlaysIds[0])
-		);
+	hoveredOverlay$: Observable<any> = this.store$.select(overlaysStateSelector)
```
Comment (yairtawil) [inline src/app/@ansyn/overlays/components/overlay-overview/overlay-overview.component.ts]: create `selector` for `hoveredOverlay`

File: src/app/@ansyn/ansyn/app-effects/effects/overlays.app.effects.ts (hunk 2 above)
Comment (ramtob) [reply src/app/@ansyn/ansyn/app-effects/effects/overlays.app.effects.ts]: done

File: src/app/@ansyn/overlays/components/overlay-overview/overlay-overview.component.ts (hunk 3 above)
Comment (ramtob) [reply src/app/@ansyn/overlays/components/overlay-overview/overlay-overview.component.ts]: done

**Pull request merged**